hypothesis-python/src/hypothesis/strategies.py

```diff
@@ -1901,89 +1915,76 @@ def runner(default=not_set):
-        def __repr__(self):
-            return 'data(...)'
+    def do_draw(self, data):
+        data.can_reproduce_example_from_repr = False
 
-        def draw(self, strategy, label=None):
-            result = self.data.draw(strategy)
-            self.count += 1
-            if label is not None:
-                note('Draw %d (%s): %r' % (self.count, label, result))
-            else:
-                note('Draw %d: %r' % (self.count, result))
-            return result
+        if not hasattr(data, 'hypothesis_shared_data_strategy'):
+            data.hypothesis_shared_data_strategy = DataObject(data)
+        return data.hypothesis_shared_data_strategy
 
-    class DataStrategy(SearchStrategy):
-        supports_find = False
+    def __repr__(self):
+        return 'data()'
 
-        def do_draw(self, data):
-            data.can_reproduce_example_from_repr = False
+    def map(self, f):
+        self.__not_a_first_class_strategy('map')
 
-            if not hasattr(data, 'hypothesis_shared_data_strategy'):
-                data.hypothesis_shared_data_strategy = DataObject(data)
-            return data.hypothesis_shared_data_strategy
+    def filter(self, f):
+        self.__not_a_first_class_strategy('filter')
+
+    def flatmap(self, f):
+        self.__not_a_first_class_strategy('flatmap')
 
-        def __repr__(self):
-            return 'data()'
+    def example(self):
+        self.__not_a_first_class_strategy('example')
+
+    def __not_a_first_class_strategy(self, name):
+        raise InvalidArgument((
+            'Cannot call %s on a DataStrategy. You should probably be '
+            "using @composite for whatever it is you're trying to do."
+        ) % (name,))
 
-        def map(self, f):
-            self.__not_a_first_class_strategy('map')
 
-        def filter(self, f):
-            self.__not_a_first_class_strategy('filter')
+@cacheable
+def data():
+    # type: () -> SearchStrategy[Any]
+    """This isn't really a normal strategy, but instead gives you an object
+    which can be used to draw data interactively from other strategies.
 
-        def flatmap(self, f):
-            self.__not_a_first_class_strategy('flatmap')
+    It can only be used within :func:`@given <hypothesis.given>`, not
+    :func:`find() <hypothesis.find>`. This is because the lifetime
+    of the object cannot outlast the test body.
 
-        def example(self):
-            self.__not_a_first_class_strategy('example')
+    See :ref:`the rest of the documentation <interactive-draw>` for more
+    complete information.
 
-        def __not_a_first_class_strategy(self, name):
-            raise InvalidArgument((
-                'Cannot call %s on a DataStrategy. You should probably be '
-                "using @composite for whatever it is you're trying to do."
-            ) % (name,))
+    Examples from this strategy do not shrink (because there is only one),
+    but the result of calls to each draw() call shrink as they normally would.
```

Comment:
> calls to each draw() call

Unclear fragment

Reply:
I'm not really sure what you mean. It reads fine to me. (Also changing that would would be a PR with multiple user facing changes!)

Reply:
I'd say "...the results of each draw() call shrink..."